src/paragraph/Paragraph.story.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+
+import { Paragraph } from './index'
+import { BaseSection as Section } from '../layout/section/baseSection'
+
+<Meta title="Paragraph" />
+
+# **Paragraph**
+
+<Preview>
+  <Story name="Long text">
+    <Section>
```

Comment:
Please don't add Section component here since it's not necessary on the Paragraph API.
We want to maintain focus on the component itself rather than the presentation on Storybook